tests/test_travis2docker.py

```diff
@@ -106,6 +106,20 @@ def test_main():
         dkr_content = f_dkr.read()
         assert 'VARIABLE_INCLUDE_2="value include 2"' in dkr_content
 
+    # Tests that, when specified, the postgresql key sets
+    # automatically the environment variable $PSQL_VERSION
+    example = os.path.join(dirname_example, 'example_5.yml')
+    sys.argv = argv + ['--travis-yml-path', example]
+    scripts = main()
+    assert len(scripts) == 2, 'Scripts returned should be 2 for %s' % example
```

Comment:
The good practice is use [unittest methods](https://docs.python.org/2/library/unittest.html#test-cases) like as `self.assertLen(scripts, 2, "Scripts...")`

Reply:
What about my last comments?

Reply:
In this case it's not possible, because these tests are not an inheritance  from the class `unittest.TestCase`.

Reply:
WTF Im not using unittests!!!